packages/legacy-app/package.json

```diff
@@ -100,7 +100,7 @@
 		"clean": "rimraf dist",
 		"lint": "standard 'app/**/*.js'",
 		"lint:fix": "npm run lint -- --fix",
-		"test": "karma start --single-run",
+		
```

Comment:
Why is this removed in the legacy app? Shouldn't we still be able to run it's tests?

Reply:
I have fixed the karma conf and some failing test and enabled this script to run